### ISSUE TITLE
Run All Suites Button

### ISSUE THREAD

**Issue by cawoodm** [SeleniumHQ/selenium-ide#428]
## 🚀 Feature Proposal

There should be a "Run All Suites" button which runs all test suites. Ideally test suits should be drag ordered and not alphabetic as today.

## Motivation

There is currently no way to run all test suites today and this is often required.

## Example

Each .side file contains all tests for an application and we want to run all of them. Ideally in a specified order.

**Comment by sporthcophcop** [comment]
You can steer the order of the tests by simply adding a number in front of it in the test section of the plugin. (Yes it is a work around.) 
![afbeelding](https://user-images.githubusercontent.com/42733905/46525648-7a70f580-c88c-11e8-8fae-87197650a313.png)
With the current version of the plug in: 3.4.0 it is possible to order test within a test suite.
The tests order within the suite seems to be the run order.

**Comment by corevo** [comment]
@sporthcophcop You can order tests by placing them inside suites, you can drag and drop them into order.

**Comment by corevo** [comment]
![oct-05-2018 12-04-08](https://user-images.githubusercontent.com/888886/46526434-e8bab580-c896-11e8-9cc5-d9f509832c94.gif)

**Comment by sporthcophcop** [comment]
@corevo Hi. Thats exactly what I tried to indicate with my English. ("it is possible to order test within a test suite") Sorry for the missing s to indicate tests.

**Comment by cawoodm** [comment]
Guys, my proposal is a run all suites button where the order of the test suites is defined by the user. What you've shown is how to order test.

**Comment by cawoodm** [comment]
> A couple of thoughts:
> 
> 1. Have you had a look at [the command-line runner](https://github.com/SeleniumHQ/selenium-ide/tree/master/packages/selenium-side-runner)?
> 2. What about a `Run All Tests` button instead?

Yes, we are using our own .side runner in PowerShell. Here we can specify which tests or suites to run in which order but there is no "Run all suites" button in the actual IDE and no "run all suites" option.

**Comment by felipeccastro** [comment]
+1 Running all test suites is what I need too, instead of simply running all tests. There are several tests I'm reusing across different test suites (so they need to run more than once), and they always rely on the order they have been defined in the test suite, so it wouldn't work the same if I'd just had the option to run all tests.

**Comment by dailytabs** [comment]
+1 My QA just came to me today and was very confused by the fact that he cannot run all suites, with the tests running in the defined (by drag-n-drop) order.
